### ISSUE TITLE
AddressComplete: No data available for Heard Island and Mcdonald Islands 

### ISSUE THREAD

**Issue by samsadasivan** [cds-snc/platform-forms-client#4602]

**Comment by ShadeWyrm** [comment]
@samsadasivan - Can you expand on this with example addresses that didn't work?

**Comment by samsadasivan** [comment]
Yes! Basically when I select this country, then I go to the street address -> Nothing populates in the street address area. I will add a video!

**Comment by ShadeWyrm** [comment]
Looks there are no Addresses available in the API at all for these countries. Closing this as it matches the way the current address complete works on their site.
